native/src/utils/PushNotificationsManager.ts

```diff
@@ -108,7 +108,6 @@ export const useForegroundPushNotificationListener = ({
         if (mounted) {
           // The CMS needs some time until the push notification is available in the API response
           setTimeout(() => {
-            // TODO #2140: Uncomment and improve snackbar
```

Comment:
Uh, the comment's still there though?

Reply:
Yes, but it should be done in #2140 where this is described/mentioned. So I would just leave the comment.